reference/javascript_3x/APIReference.md

```diff
@@ -429,7 +429,7 @@ ADB.Media.trackEvent(event, info, contextData);
 
 #### updatePlayhead
 
-Provide current media playhead to media tracker. For accurate tracking, call this method whenever playhead changes during playback.
+Provides a media tracker with the current media playhead. For accurate tracking, call this method multiple times when the playhead changes. If the player does not notify playhead changes, call this method once every second with the most recent playhead.
```

Comment:
Provides the current media playhead to the media tracker instance. For accurate tracking, call this method everytime the playhead changes. If the player does not notify playhead changes, call this method once every second with the most recent playhead.